conjure-java-core/src/integrationInput/java/com/palantir/product/ListExample.java

```diff
@@ -189,19 +189,19 @@ public Builder from(ListExample other) {
         }
 
         @JsonSetter(value = "items", nulls = Nulls.SKIP, contentNulls = Nulls.FAIL)
-        public Builder items(@Safe @Nonnull Iterable<String> items) {
+        public Builder items(@Nonnull Iterable<String> items) {
```

Comment:
weird.. need to investigate

Reply:
Hmm, I suspect `widenParameterIfPossible` may need to be updated to retain safety info